src/Microsoft.ML.Transforms/Text/TextTransform.cs

```diff
@@ -336,22 +336,17 @@ public ITransformer Fit(IDataView input)
 
             if (tparams.NeedsWordTokenizationTransform)
             {
```

Comment:
can you fix the spacing here? It looks like this code should be moved to the left.